freqtrade/freqtradebot.py

```diff
@@ -54,16 +54,19 @@ def __init__(self, config: Dict[str, Any]) -> None:
 
         self._heartbeat_msg = 0
 
-        self.heartbeat_interval = self.config.get('internals', {}).get('heartbeat_interval', 60)
+        self.heartbeat_interval = self.config.get("internals", {}).get(
+            "heartbeat_interval", 60)
```

Comment:
I'm not a huge fan of this change.

I don't mind which quotes we're using necessarily (although i don't see a huge reason to change where we got them - but we've got default settings (with line-lenght to 100 - as pointed out by @hroff-1902).
 This seems to reduce the line-length to 80ish (probably 88, judging from your screenshot) - causing a lot of unnecessary linebreaks (like this one - but there's a few others before even getting to line 100)

Reply:
I'll fix it to 100 characters per line. Then we can take a look at it again.